packages/framework/src/Model/MultidomainEntityClassProvider.php

```diff
@@ -43,6 +44,7 @@ public function getManualMultidomainEntitiesNames(): array
             ProductDomain::class,
             TransportDomain::class,
             CountryDomain::class,
+            SeoPageDomain::class,
```

Comment:
I have got a bad feeling now, how did manage to not forget about this dark place? I am sure I have not added here some entities (`BlogArticleDomain`?) in my implementations. However, I do not think you need to define each multidomain entity here, or? Let's discuss this together please

Reply:
All is fine as we discussed